packages/plugin/core/src/lib/connector.ts

```diff
@@ -34,19 +35,31 @@ export function isPluginMessage(message: any): message is Message {
  */
 export function connectClient(connector: ClientConnector, client: PluginClient = new PluginClient()) {
   let isLoaded = false
+  let account
+  connector.on(async ({ action, key, name, payload, id, signature, verifier, requestInfo, error }) => {
+    if (!signature && isLoaded) {
+      throw new Error('signature should have been set')
+    }
+    if (signature) {
+      const sign = fromRpcSig(signature)
+      const address = ecrecover(toBuffer(verifier), sign.v, sign.r, sign.s)
```

Comment:
This returns public key and it is compared with address: https://github.com/ethereumjs/ethereumjs-util/blob/master/docs/modules/_signature_.md#const-ecrecover